game/src/actors/Enemy.gd

```diff
@@ -0,0 +1,26 @@
+extends "res://src/actors/PlatformActor.gd"
```

Comment:
So I renamed this to `Actor.gd` because Platform implies something environment-related so...